Cabal/Distribution/Make.hs

```diff
@@ -126,7 +131,9 @@ defaultMainHelper args =
       ,unregisterCommand      `commandAddAction` unregisterAction
       ]
 
-configureAction :: ConfigFlags -> [String] -> IO ()
+type Action flags = flags -> [String] -> Prelude.IO ()
```

Comment:
This changes (WithCallStack)`IO` to `Prelude.IO`, and I guess this is the place where we'd like the `CallStack`s to propagate.